src/components/TimeSeriesCharts.tsx

```diff
@@ -56,21 +56,31 @@ class TimeSeriesCharts extends React.Component<TimeSeriesChartsProps> {
             removeTimeSeriesGroup, showPointsOnly, showErrorBars,
             placeInfos
         } = this.props;
-        const charts = timeSeriesGroups.map(timeSeriesGroup => (
-            <TimeSeriesChart
-                key={timeSeriesGroup.id}
-                locale={locale}
-                timeSeriesGroup={timeSeriesGroup}
-                selectedTime={selectedTime}
-                selectedTimeRange={selectedTimeRange}
-                dataTimeRange={dataTimeRange}
-                selectTime={selectTime}
-                selectTimeRange={selectTimeRange}
-                removeTimeSeriesGroup={removeTimeSeriesGroup}
-                showPointsOnly={showPointsOnly}
-                showErrorBars={showErrorBars}
-                placeInfos={placeInfos}
-            />)
+        const charts = timeSeriesGroups.map(timeSeriesGroup => {
+                let completed = [];
```

Comment:
Use `const completed = timeSeriesGroup.timeSeriesArray.map(...)` instead for readability.